test/box-luatest/alter_with_compression_test.lua

```diff
@@ -0,0 +1,275 @@
+local t = require('luatest')
+
+box.cfg{}
```

Comment:
The luatest instance must not call `box.cfg` AFAIU. You should create another instance and wrap all checks in `exec()` (it's okay to call `t.assert` from `exec`).

Reply:
fixed